ocs_ci/ocs/resources/mcg.py

```diff
@@ -117,7 +117,10 @@ def __init__(self):
 
         self.s3_client = self.s3_resource.meta.client
 
-        if config.ENV_DATA['platform'].lower() == 'aws':
+        if (
+            config.ENV_DATA['platform'].lower() == 'aws'
+            and kwargs.get('create_aws_creds')
```

Comment:
what if `create_aws_creds` doesn't exist in kwargs? Please add a default:

```python
kwargs.get('create_aws_creds', 'default value')
```

Reply:
`.get()`returns `None` by default, which will be evaluated as `False` in this scenario - so this is working as intended

Reply:
ack, thanks